tv_apps/dashboard/style/dashboard.css

```diff
@@ -140,3 +140,33 @@ iframe {
   width: 100%;
   height: 100%;
 }
+
+/* Intro animation */
+.active #bottom-panel {
+  transform-origin: 50vw calc(16rem - 50vh);
```

Comment:
It tries to set the origin to center of the screen, but we have to get that from point of each panel.
I'll add some comments to describe it.